jwala-common/src/test/java/com/cerner/jwala/commandprocessor/impl/jsch/JschCommandProcessorImplTest.java

```diff
@@ -1,67 +0,0 @@
-package com.cerner.jwala.commandprocessor.impl.jsch;
```

Comment:
This should be in another branch. The concept of branching is so that changes to a particular goal are grouped together and easier to digest.